.github/workflows/build_and_test.yml

```diff
@@ -455,7 +455,7 @@ jobs:
         java-version: 11
     - name: Build with Maven
       run: |
-        export MAVEN_OPTS="-Xss16m -Xmx2g -XX:ReservedCodeCacheSize=1g -Dorg.slf4j.simpleLogger.defaultLogLevel=WARN"
+        export MAVEN_OPTS="-Xss64m -Xmx2g -XX:ReservedCodeCacheSize=1g -Dorg.slf4j.simpleLogger.defaultLogLevel=WARN"
```

Comment:
@gengliangwang, sorry but since you're here, would you mind add this option to https://github.com/apache/spark/blob/master/appveyor.yml#L52 too?

Reply:
and actually I think we can remove `-Xss64m` here because it's added into POM explicitly at https://github.com/apache/spark/commit/b59d5ab0604a5794f316337685af0002af67c991#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8